src/services/scriptingservice.cpp

```diff
@@ -2012,6 +2012,56 @@ QVariant ScriptingService::getApplicationSettingsVariable(
     return settings.value(key, defaultValue);
 }
 
+/**
+ * Fetches all subfolder ids for subfolders that are children of parentId
+ *
+ * @param parentId {int} id of parent subfolder
+ * @return QList<int> list of subfolder ids
+ */
+QList<int> ScriptingService::fetchSubFoldersByParentId(int parentId)
```

Comment:
This actually doesn't return NoteSubFolders, but only ids, doesn't it?
I generally would prefer it more if there actually would be `NoteSubFolderApi` class (like for notes and tags) with proper methods that return actual objects.

Reply:
Asked you regarding this on IRC, written to @pbek (probably not the correct username?) :)

Reply:
Ok, @pbek so we should drop this review and implement the `NoteSubFolderApi` instead, is that right ?

Reply:
Yes, please.

Reply:
> Asked you regarding this on IRC, written to @pbek (probably not the correct username?) :)

It is, but I didn't get any notifications. 😬  In the `#qownnotes` channel your messages would have been bridged to Telegram.